code/processes/tickerlogreplay.q

```diff
@@ -231,7 +246,7 @@ realupd:{[f;t;x]
 	}[.replay.upd]
 
 // amend the upd function to filter based on the table list
-if[not tablelist~enlist `all; realupd:{[f;t;x] if[t in .replay.tablestoreplay; f[t;x]]}[realupd]]
+if[(not tablelist~enlist `all) and not segmentedmode; realupd:{[f;t;x] if[t in .replay.tablestoreplay; f[t;x]]}[realupd]]
```

Comment:
please address this

Reply:
Hi Jonny, there is code to filter out the stpmeta table in the realupd function on line 220. Is that what you are referring to? Or are you asking if there is code that filters out the stpmeta table for replaying when the stp is not in segmented mode?

Reply:
No. What I mean is, if the tickerlogreplay is in segmented mode, then the stp meta table should containing the information on what tables each log file contains. If the replay is only for a subset of tables, then it can ignore all of the log files which don't contain any of the tables it is attempting to replay. I don't think it does that currently.